packages/examples/src/reactTs.tsx

```diff
@@ -18,7 +18,9 @@ const EditorDemo: React.FC = () => {
         htmlElement: rootElem,
         wrapperConfig: {
             serviceConfig: {
-                enableKeybindingsService: true,
+                userServices: {
+                    ...getKeybindingsServiceOverride()
```

Comment:
Just a question, is there a risk of services overriding properties contributed from each other?

Reply:
You can a service already specified before being initialized, but the parameters a unique to each single service.